tracer/dependabot/Datadog.Dependabot.Integrations.csproj

```diff
@@ -78,18 +78,18 @@
 
     <!-- Integration: System.Net.Http.WinHttpHandler -->
     <!-- Assembly: System.Net.Http.WinHttpHandler -->
-    <!-- Latest package https://www.nuget.org/packages/System.Net.Http.WinHttpHandler/6.0.1 -->
+    <!-- Latest package https://www.nuget.org/packages/System.Net.Http.WinHttpHandler/7.0.0 -->
     <PackageReference Include="System.Net.Http.WinHttpHandler" Version="6.0.1" />
 
     <!-- Integration: Microsoft.Extensions.Logging -->
     <!-- Assembly: Microsoft.Extensions.Logging -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging/6.0.0 -->
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging/7.0.0 -->
     <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
 
     <!-- Integration: Microsoft.Extensions.Logging.Abstractions -->
     <!-- Assembly: Microsoft.Extensions.Logging.Abstractions -->
-    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/6.0.2 -->
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.2" />
+    <!-- Latest package https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions/7.0.0 -->
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="6.0.3" />
```

Comment:
Curious why does this one and a couple others above it differ from the latest version in the comment?

Reply:
Ah, I think that's because I didn't regenerate the package versions. Done now, and that's fixed these. There may still be need packages which updating, but that'll be for a subsequent PR 🙂